ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/BraveTranslateScriptHandler.swift

```diff
@@ -91,14 +91,15 @@ class BraveTranslateScriptHandler: NSObject, TabContentScript {
     }
 
     // Processing
-    Task {
+    Task { [weak tab] in
+      guard let tab = tab else { return }
```

Comment:
You need to execute the replyHandler even if the tab dies, no?

Reply:
Technically it wont' cause an issue as it'll just throw a Javascript exception `unhandled promise`. But I fixed it :)